frontend/src/metabase/entities/collections/getExpandedCollectionsById.js

```diff
@@ -28,7 +28,6 @@ function getExpandedCollectionsById(
           : null,
       parent: null,
       children: [],
-      is_personal: c.personal_owner_id != null,
```

Comment:
This property is hydrated from BE, so we need to remove the FE counterpart. I didn't know this exists before and it will be in conflict with the property from BE, so I need to remove this.